docs/contracts/v4/concepts/02-pool-manager-and-pool.mdx

```diff
@@ -0,0 +1,175 @@
+---
+id: pool-manager-and-pool-initialization
+title: Pool Manager and Pool Initialization
+sidebar_position: 1
+---
+
+# Architecture
+In Uniswap v3, each pool has its own contract instance, which makes making pools and doing swaps in multiple pools costly
+
+![High Level Architecture](./images/01_Pool_Initialization/HighLevelArchitecture.png)
+
+In v4, all pools are kept in a single contract instance, saving a lot of important gas because tokens don’t have to move between different pool contracts during swaps. Early calculations say that v4 will make the gas cost of creating pools 99% less. Hooks offer unlimited choices and the single contract lets you easily move through all these choices.
+
+
+
+![Detailed Architecture](./images/01_Pool_Initialization/DetailedArchitecture.png)
+
+This Singleton design is improved by a new "flash accounting" method. Instead of moving assets in and out of pools after each swap in v3, this method only moves the net balances. This means the system is a lot more efficient and saves even more gas in Uniswap v4.
+
+Because of the efficiency of the Singleton contract and flash accounting, there is no need to limit fee tiers. People who create pools can choose them to be most competitive or change them with a dynamic fee hook. v4 also supports native ETH again, which helps save more gas.
+
+# PoolManager
+To understand the major parts of the PoolManager, let's look at the the main interface it implements: `IPoolManager.sol`.
+
+https://github.com/Uniswap/v4-core/blob/main/src/interfaces/IPoolManager.sol
+
+Here’s a brief overview of the interface.
+
+- It inherits from `IFees` and `IERC1155`, combining fee management and token management functionalities.
+- It then defines errors to handle various exceptional conditions like max currencies touched, currency not settled, and improper function caller.
+- It then includes the `Initialize`, `ModifyPosition`, and `Swap` events which get emitted during respective operations, helping in tracking state changes.
+- It then has the `initialize` function which is used to initialize a new pool.
+- Various functions are declared to manage and query pool states like `getSlot0`, `getLiquidity`, and `getPosition`.
+- There are functions to manage liquidity positions in pools such as `modifyPosition` and `swap`, which facilitate liquidity provision and token swaps.
+- It includes mechanisms to manage pool locks and concurrency through functions like `lock`.
+- Functions like `mint`, `take`, and `settle` manage user balances, allowing users to interact with their funds within pools.
+- It allows external contracts to access pool state through the `extsload` function, facilitating integration with other contracts or interfaces.
+- Custom structs like `ModifyPositionParams` and `SwapParams` are declared to bundle parameters neatly, improving code readability and usability.
```

Comment:
We should take this out. It doesnt provide useful information, and will be confusing to devs.

Instead, I think we should just highlight the main lifecycle functions: `initialize, swap, modifyPosition`

and then balance functions `mint, burn, take, and settle`. I would also note that `mint/burn` applies to the 6909 claims, and not LP receipt tokens